lib/ckb/lock_handlers/anyone_can_pay_handler.rb

```diff
@@ -0,0 +1,30 @@
+module CKB
+  module LockHandlers
+    class AnyoneCanPayHandler < SingleSignHandler
+      attr_reader :anyone_can_pay_cell_dep
+
+      def initialize(anyone_can_pay_tx_hash = "0xeba5986c50e7e8215c2acd6501150cab1346119f42f4d7870bfadd5bfdf7fe57", index = 0)
```

Comment:
Maybe set `anyone_can_pay_tx_hash ` will be more better ?

Reply:
I decided to change to manually set lock and type that are not built into the system